Python/setup.py

```diff
@@ -33,5 +33,5 @@
        package_dir = {'pywarpx':'pywarpx'},
        description = """Wrapper of WarpX""",
        package_data = package_data,
-       install_requires=['picmistandard', 'periodictable']
+       install_requires=['numpy', 'picmistandard', 'periodictable']
```

Comment:
is `mpi4py` required by default?

Reply:
mpi4py is optional
It will be imported if available, otherwise the code will assume serial.